libs/application/templates/party-letter/src/fields/Recommendations/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FC, useState } from 'react'
+import { FieldBaseProps } from '@island.is/application/core'
+import { Box, Text, Input, Checkbox } from '@island.is/island-ui/core'
+import CopyLink from './CopyLink'
+import RecommendationTable from './RecommendationTable'
+
+const SIGNATURES = [
```

Comment:
do you want to keep that here?

Reply:
This is a WIP and will be populated with correct data via another PR, this is just here to display correct UI